runtime/relay-encoder/src/kusama.rs

```diff
@@ -80,6 +80,8 @@ pub enum HrmpCall {
 	AcceptOpenChannel(ParaId),
 	#[codec(index = 2u8)]
 	CloseChannel(HrmpChannelId),
+	#[codec(index = 3u8)]
```

Comment:
Cancel is index 06, not 03. We should modify it and add unitest in each of the files